bash2048.sh

```diff
@@ -262,6 +264,70 @@ function reload_game {
   index_max=board_size-1
 }
 
+# print the content of the highscores file
+function show_highscores {
+    output="\nHighscore Name Date\n\n"
+    while read line; do
+        output="${output}${line}\n"
+    done < "${highscore_file}"
+    printf "${output}" | column -t
```

Comment:
I tried to avoid commands like column because people play this on weird devices and they sometimes lack these utilities, so it would be preferable to do the formatting here in bash instead of using column

Reply:
Done :+1: